onnxruntime/test/debug_node_inputs_outputs/debug_node_inputs_outputs_utils_test.cc

```diff
@@ -27,7 +27,7 @@ void VerifyTensorProtoFileData(const PathString& tensor_proto_path, gsl::span<co
 
   std::vector<T> actual_data{};
   actual_data.resize(expected_data.size());
-  ASSERT_STATUS_OK(utils::UnpackTensor(tensor_proto, Path{}, actual_data.data(), actual_data.size()));
+  ASSERT_STATUS_OK(utils::UnpackTensor(tensor_proto, std::filesystem::path(), actual_data.data(), actual_data.size()));
```

Comment:
std::filesysstem::path(tensor_proto_path) ?

Reply:
The path is model path, where the model file lives. Here we do not have a model.